src/includes/getting-started-config/dotnet.aspnet.mdx

```diff
@@ -0,0 +1,61 @@
+Add Sentry to `Global.asax.cs` as shown here:
```

Comment:
Is this file being used anywhere? Seems we're overriding the index.mdx already so this include isn't used.

It's even live already the global asax:
https://docs.sentry.io/platforms/dotnet/guides/aspnet/

Reply:
should i delete the file?